mitmproxy/flowfilter.py

```diff
@@ -492,25 +493,42 @@ def _make():
 def parse(s):
     # type: (str) -> TFilter
     try:
-        filt = bnf.parseString(s, parseAll=True)[0]
-        filt.pattern = s
-        return filt
+        filter = bnf.parseString(s, parseAll=True)[0]
```

Comment:
As tempting as it is, we don't want to override builtins :wink: 
filtexpr, filter_function, flowfilter, ... ?